### DISCUSSION TITLE
Add PDF documentation in your application!

### DISCUSSION THREAD

**Discussion by Akascape** [TomSchimansky/CustomTkinter#1677] [Showcase]
## [CTkPDFViewer](https://github.com/Akascape/CTkPDFViewer)

If you are making a complex application, then you have to create documentation for the users. It is better if you provide an offline documentation in form of a PDF file.

With CTkPDFViewer, you can embed that PDF file in your application so that users can view it immediately.

![example](https://github.com/Akascape/CTkPDFViewer/assets/89206401/1324243e-da47-4bd8-af46-cded53cb7b51)

**Comment by MaxNiark** [comment]
Thank bro !
